pom.xml

```diff
@@ -1821,12 +1825,17 @@
           </exclusion>
         </exclusions>
       </dependency>
-      <!-- Hive 2.3 need this to init Hive's FunctionRegistry -->
+      <dependency>
+        <groupId>org.codehaus.jackson</groupId>
+        <artifactId>jackson-core-asl</artifactId>
+        <version>${codehaus.jackson.version}</version>
+        <scope>${hadoop.deps.scope}</scope>
```

Comment:
Hadoop 3.3.4 still need this: https://issues.apache.org/jira/browse/HADOOP-13332